### DISCUSSION TITLE
How to start a lima VM non-interactively?

### DISCUSSION THREAD

**Discussion by ChillarAnand** [lima-vm/lima#1420] [Q&A] [answered]
When starting a VM, lima prompts users with the following options.

<img width="1261" alt="Screenshot 2023-03-15 at 09 46 05" src="https://user-images.githubusercontent.com/4463796/225205138-9c990ddb-9099-4456-a970-6120b2af31f2.png">

How to disable this prompt and use the default config? I am writing scripts to automate provisioning VMs with lima.

**Comment by AkihiroSuda** [comment, marked as answer]
`limactl start --tty=false`

**Reply by TravisDart** [reply]
This also works with `limactl create`. E.g.

```
limactl create --name default2 --tty=false --cpus=8 --memory=16 --vm-type vz --mount-type=virtiofs --network=vzNAT --mount . --mount-writable template://docker-rootful
```
